source/docs/software/frc-glossary.rst

```diff
@@ -53,9 +89,39 @@ FRC Glossary
    NetworkTables
       A way to communicate key / value pairs of data between programs.
 
+   mass
+      the amount of matter in a physical object. Objects with more mass will resist changes in motion more than objects with less mass. See `mass <https://en.wikipedia.org/wiki/Mass>`_ on wikipedia for more info.
+
+   moment of inertia
+      The property of an object that describes both how much mass it has, and how that mass is distributed relative to a certain axis of rotation. Objects with higher moments of inertia resist changes in rotational motion more than objects with lower moments of inertia. Increasing the moment of inertia is accomplished by adding more mass, or moving the mass further away from the axis of rotation. See `moment of inertia <https://en.wikipedia.org/wiki/Moment_of_inertia>`_ on wikipedia for more info.
+
+   permanent-magnet DC motor
+      The classification of all legal motors for the FIRST robotics competition. This type of motor takes direct current as input, and uses it to create a magnetic field. In turn, this magnetic field interacts with a physical magnet to create a force that turns the output shaft. Electrical ("brushless") or mechanical ("brushed") means are used to ensure the electrically-generated magnetic field always points in a direction that creates forces when it interacts with the physical magnet, even as the motor's shaft rotates. See `permanent-magnet motor <https://en.wikipedia.org/wiki/Brushed_DC_electric_motor#Permanent-magnet_motors>`__ on wikipedia for more info.
+
+   retro-reflection
+      The property of reflecting incoming light back at the same angle it came in at, rather than an incident angle (like a mirror), absorbing it, or scattering it. Most FRC vision processing targets are retro-reflective. See `retroreflector <https://en.wikipedia.org/wiki/Retroreflector>`_ on wikipedia for more information.
+
+   recursive composition
+      A software term for when an object is made up of many objects, including itself. For example, a command group may contain one or more command groups. See `recursive composition <https://en.wikipedia.org/wiki/Object_composition#Recursive_composition>`__ on wikipedia for more info.
```

Comment:
"A type of :term:`composition` in which the composite object may contain components of the same type as itself."